bigchaindb/tendermint/commands.py

```diff
@@ -61,6 +59,9 @@ def start():
                                  args=(exchange.get_publisher_queue(),))
     p_websocket_client.start()
 
+    p_exchange = Process(name='exchange', target=exchange.run)
```

Comment:
Why moving it after the websocket initialization and start?

Reply:
The `exachge` object is somehow not updated after `exchange.get_subscriber_queue(EventTypes.BLOCK_VALID),)` is executed.

Reply:
OK, I got what the problem is, and thanks a lot for debugging :clap:. I'll create a new PR after merging this one.